jre/java/java/util/stream/DoubleStream.java

```diff
@@ -202,6 +202,39 @@ static DoubleStream of(double t) {
 
   DoubleStream distinct();
 
+  default DoubleStream dropWhile(DoublePredicate predicate) {
+    Spliterator.OfDouble prev = spliterator();
+    Spliterator.OfDouble spliterator =
+        new Spliterators.AbstractDoubleSpliterator(prev.estimateSize(),
+                prev.characteristics() & ~(Spliterator.SIZED | Spliterator.SUBSIZED)) {
+          private boolean drop = true;
+          private boolean found;
+
+          @Override
+          public boolean tryAdvance(DoubleConsumer action) {
```

Comment:
I think it is easier to read if you do:

if (!dropping) {
  return prev.tryAdvance(action);
}

// Drop items until we find one that matches.
....